docs/_platforms/building_jedi_code_on_discover_sles15.md

```diff
@@ -0,0 +1,59 @@
+# Building JEDI code on Discover (SLES 15)
+
+1) Load the JEDI spack Python modules (These modules will load Python version 3.10):
+
+``` bash
+module purge
+module use /discover/swdev/jcsda/spack-stack/scu17/modulefiles
+module load ecflow/5.11.4
+
+module use /gpfsm/dswdev/jcsda/spack-stack/scu17/spack-stack-1.7.0/envs/ue-intel-2021.10.0/install/modulefiles/Core
+module load stack-intel/2021.10.0
+module load stack-intel-oneapi-mpi/2021.10.0
+module load stack-python/3.10.13
+module load git-lfs/3.4.0
+module load py-pip/23.1.2
+```
```

Comment:
Surprising number of module setup steps. Is there something stopping us from creating a file in `advda/swell/jedi_modules` to source from instead?

Reply:
On SLES12, miniconda was handling the modules and pip under the hood. Spack-stack team mentioned they won't support miniconda no more so now we have to load these modules individually. But to your point, sure, I can just add a lightweight `jedi_modules` SLES15 file because `jedi_bundle` has far few more dependencies.